pyvista/core/pointset.py

```diff
@@ -2237,11 +2237,11 @@ def _from_arrays(self, dims: Sequence, corners: Sequence) -> None:
 
         Parameters
         ----------
-        dims : Sequence
+        dims : sequence[int]
             A sequence of integers with shape (3,) containing the
             topological dimensions of the grid.
 
-        corners : Sequence
+        corners : sequence[floats]
             A sequence of floats with shape (number of corners, 3)
             containing the coordinates of the corner points.
```

Comment:
I'd also make anything with "shape" an `array_like`
```suggestion
        corners : array_like[floats]
            A sequence of floats with shape (number of corners, 3)
            containing the coordinates of the corner points.
```